tests/sympc/protocol/falcon/falcon_test.py

```diff
@@ -108,17 +112,21 @@ def test_mul_private_exception_nothreeparties(get_clients, parties):
         tensor1 * tensor2
 
```

Comment:
I was wondering if there was any way you could test if an exception was being raised in case of malicious share values.

Reply:
I thought of the scenario, a place where we could make it raise is, modify  the triples after generation initially, this would raise the error, it has to be done locally by the parties, I tried to modify crypto store values after generation, was not able to do so,not sure if orchestrator could modify the values of the parties? @gmuraru ,is there any way for orchestrator to modify crypto store values.
Or there might some better way to raise that error, any suggestions?

Reply:
I haven't been able to think of any way yet.

Reply:
I have added a test for the same.